binders/difido-testng/src/main/java/il/co/topq/difido/config/AbstractPropertiesConfigFile.java

```diff
@@ -79,22 +54,34 @@ private void initDefaultProperties() {
 			log.warning("Failed to write Difido properties to file");
 		}
 	}
-
-	public String getPropertyAsString(RemoteDifidoOptions option) {
```

Comment:
Ok, you can change it, but remember that there are two special cases like Map and List that require splitting.